src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -564,6 +564,23 @@ class spell_mage_firestarter_dots : public AuraScript
     }
 };
 
+// 205029 - Flame On
+class spell_mage_flame_on : public AuraScript
+{
+   PrepareAuraScript(spell_mage_flame_on);
+
+   void CalculateAmount(AuraEffect const* /*aurEff*/, int32& amount, bool& canBeRecalculated)
+   {
+       canBeRecalculated = false;
+       amount = -16;
```

Comment:
if its just set to a constant we likely may use SpellMgr for this. Also we may calculate it directly `effect3 basepoints / rechargeTime * 100`

Reply:
Eeeh its more complicated than that.
Cooldown of Fire Blast is reduced by haste and the talent is then supposed to reduce it by flat 2 seconds (that means this aura is supposed to update value with stats (haste) changes)

Reply:
shouldn't we already take that into account by using the current rechargetime for the calculation?

Reply:
no, this is fine